app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,34 @@
+class Api::V1::UsersController < Api::V1::BaseController
+  before_action :authenticate!, only: %i(change_rival score_viewer)
+
+  def status
+    render json: { status: current_user.try(:iidxid) }
+  end
+
+  def me
+    render json: { current_user: current_user.try(:schema) }
+  end
+
+  def change_rival
```

Comment:
__[rails_best_practices]__
* move model logic into model (target_user use_count > 4) [:ref](http://rails-bestpractices.com/posts/2010/07/21/move-model-logic-into-the-model/)